src/components/accordian/Accordian.tsx

```diff
@@ -17,15 +18,19 @@ type TProps = {
 
 export const Accordian = ({
   title,
-  startOpen = false,
+  open = false,
   overflowOverride,
   fixedHeight = "300px",
   children,
   showFade = "false",
   headContent,
   ...props
 }: TProps) => {
-  const [isOpen, setIsOpen] = useState(startOpen);
+  const [isOpen, setIsOpen] = useState(open);
```

Comment:
Concern: If the open state is now being controlled by a prop, why is internal state needed if it will mirror the prop value?

Reply:
It isn't solely controlled by the prop, I've made this change so that we can optionally control the accordion from outside of itself. It still has the ability to control itself from within the component.